utils/schemeParser-utils.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+
+const SLPSDK = require("slp-sdk");
+const SLP = new SLPSDK();
+
+const BITBOX = require("bitbox-sdk").BITBOX;
+const bitbox = new BITBOX();
+
+const tokenIdRegex = /^([A-Fa-f0-9]{2}){32,32}$/;
+
+const parseAddress = (address: string) => {
+  let type = getType(address);
+
+  if (type === "cashaddr") {
+    try {
+      checkIsValid("cashaddr", address);
+    } catch (error) {
+      throw new Error("invalid address");
+    }
+    return bitbox.Address.toCashAddress(address);
+  } else if (type === "slpaddr") {
+    try {
+      checkIsValid("slpaddr", address);
+    } catch (error) {
+      throw new Error("invalid address");
+    }
+    return SLP.Address.toSLPAddress(address);
+  }
+  return address;
+};
+
+const parseSLP = (params: object, tokensById: object) => {
```

Comment:
better flow types please, otherwise on point in having them. 
`tokendById` might look like `byId: { [tokenId: string]: TokenData },`